infra/app/function.bicep

```diff
@@ -0,0 +1,78 @@
+param name string
+param location string = ''
+param appServicePlanId string 
+param storageAccountName string = ''
+param tags object = {}
+@secure()
+param appSettings object = {}
+param serviceName string = 'function'
+param runtimeName string = 'python'
+param runtimeVersion string = '3.11'
+@secure()
+param clientKey string
+param keyVaultName string = ''
+param azureOpenAIName string = ''
+param azureCognitiveSearchName string = ''
+param formRecognizerName string = ''
+param contentSafetyName string = ''
+param useKeyVault bool
+param openAIKey string = ''
+param storageAccountKey string = ''
+param formRecognizerKey string = ''
+param searchKey string = ''
+param contentSafetyKey string = ''
+param keyVaultEndpoint string = ''
```

Comment:
Any parameter that is a key needs to be marked with "secure()" decorator.

Reply:
The keys here are the key names stored in the `KeyVault`, not the actual key values.